internal/ast/utilities.go

```diff
@@ -3033,7 +3033,7 @@ func IsJSDocSingleCommentNode(node *Node) bool {
 }
 
 func IsValidTypeOnlyAliasUseSite(useSite *Node) bool {
-	return useSite.Flags&NodeFlagsAmbient != 0 ||
+	return useSite.Flags&(NodeFlagsAmbient|NodeFlagsJSDoc) != 0 ||
```

Comment:
Kind of funny that you didn't do this in the other PR.

Reply:
Other PR?